dockstore-client/src/main/java/io/github/collaboratory/wdl/WDLClient.java

```diff
@@ -198,35 +205,58 @@ public Boolean check(File content) {
                     outputFound = true;
                 }
             }
-            //check all the required fields and give error message if it's missing
-            if (counter > 0 && wfFound && commandFound && callFound && outputFound) {
-                return true;    //this is a valid WDL file
+            // check all the required fields and give error message if it's missing
+            if (wfFound && callFound) {
+                String missingPotentiallyRequiredFields = "";
```

Comment:
You might find it easier to just have a `List<String>`, then use `String.join()`.

Reply:
Fixed!